charts/monitoring/prometheus/rules/kubermatic-seed-kubermatic.yaml

```diff
@@ -28,6 +28,15 @@ groups:
     for: 0m
     labels:
       severity: warning
+  - alert: KubermaticAddonTakesTooLongToReconcile
+    annotations:
+      message: Addon {{ $labels.addon }} in cluster {{ $labels.cluster }} has no related resources
+        created for more than 30min.
+      runbook_url: https://docs.loodse.com/kubermatic/master/monitoring/runbook/#alert-kubermaticaddonreconciliationtakestoolong
```

Comment:
I guess this also should be doc.kubermatic.com

Reply:
Yeah but this one is generated from the `src/` file :)